packages/core/src/makeResetStyles.ts

```diff
@@ -0,0 +1,35 @@
+import { resolveResetStyleRules } from './runtime/resolveResetStyleRules';
+import type { GriffelResetStyle, MakeStylesOptions } from './types';
+
+/**
+ * @internal
+ */
+export function makeResetStyles(styles: GriffelResetStyle) {
```

Comment:
worth adding a benchmark in the benchmark app ?

Reply:
Yes (#240), but once I will have a next PR with changes to `@griffel/react` out.